docs/bootcamp/electrical.md

```diff
@@ -0,0 +1,65 @@
+
+## Bootcamp Objectives
+
+- Learn how to make a PCB board.
+- Learn basic computer-aided circuit design principles.
+- Learn where to find documentation and how to understand it.
+- Learn about basic circuit design
+
+## Bootcamp Outline
+This Bootcamp has two options, both of which require a Team Leads approval of completion in order to move onto working on other tasks. The estimated time of completion for this Bootcamp is 1 week.
+
+## 1. Tracking Antenna Power Circuit
+
+### Background
+
+The tracking antenna helps to ensure our ground station keeps in contact with the plane at all times by pointing antennas at the plane, using data from the ground station. To do this, the tracking antenna has 2 servos that point the antennas, an arduino to do computations and basic control, and multiple sensors to help in calibration. 
+The power circuit allows the antenna to be powered by either a battery or from a power cable, and simplifies the wiring to the servos and arduino.
+
+### Task
+
+Your task is to design a PCB using DipTrace which will be able to take power from a battery or power cable.  If both battery and power cable are connected, it should default towards the power cable. It will have pin connections for the 2 servos (pan and tilt) and the arduino (5V, GND, and a PWM signal for each servo). To save money, design the PCB as small as possible. There will also be multiple LED indicators to show the circuit is working.
+
+The example schematic is at the bottom of this page. You can copy it directly, make some improvements, or design your own.  
+
+Note: We are not actually printing this circuit due to the cost and time constraints.
+
+###Inputs, Outputs, and LEDs
+- 3-wire PWM cable for pan servo
+- 3-wire PWM cable for tilt servo
+- 4 female pins for arduino wires (5V, GND, Pan, and Tilt)
+- JST battery connector (input power)
+- DC 5.5mm barrel jack (9-12V input power)
+- LED indicator to indicate the board is powered
+
+### Project Checklist
+- Schematic, PCB, and parts list
+- Inputs and outputs listed above
+- Circuit elements (MOSFET and Voltage regulator) that regulate power and can switch between the two sources
+- If both power elements are plugged in, it defaults to the power cable
+- Labels to define inputs, outputs, and component placements
+- Traces are adequately sized and well routed
+
+### Using DipTrace
+Go to this link and download the freeware:
+http://diptrace.com/download-diptrace/
+
+If you need help with using the CAD software, here are some resources:
+Diptrace Tutorial: http://diptrace.com/support/tutorials/
```

Comment:
A markdown link is in the format of [text of link](actual link). Start a server with `mkdocs serve` to verify the results